components/schema-migrator/migrations/director/20230314160000_add_tags_labels_and_documentation_labels.up.sql

```diff
@@ -0,0 +1,250 @@
+BEGIN;
+
+DROP VIEW IF EXISTS tenants_apps;
+DROP VIEW IF EXISTS listening_applications;
+DROP VIEW IF EXISTS api_resource_definitions;
+DROP VIEW IF EXISTS api_specifications_tenants;
+DROP VIEW IF EXISTS event_specifications_tenants;
+DROP VIEW IF EXISTS tenants_specifications;
+
+ALTER TABLE products
+    ADD COLUMN tags JSONB;
```

Comment:
You added new column/s in several tables and created the "small" views for them (e.g. `ord_tags_products`, etc). So far so good.

But you will have to re-create the "big" views that are used in ORD service as well so that you can include there the newly added columns. You have done it partially for `tenants_apps` when adding `application_namespace`. You will have to do it for:
- add `tags` in `tenants_products` view
- add `tags` in `tenants_vendors` view
- add `tags` in `tenants_apps` view
- add `tags` in `tenants_bundles` view
- add `hierarchy` and `supported_use_cases` in `tenants_apis` view
- add `hierarchy` and `supported_use_cases` in `tenants_events` view